app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
I wouldn't replace with an exception, if we move the create logic out of the model then we can lean on the `root` method already provided by `closure_tree`, since with a single root it will have equivalent behavior. If a single root is a constraint of our system, then we should define that on a validation and make it explicit. This is a good case for inclusion in `seeds.rb` in production, and for the test suite, [the advice in this discussion from a few PRs back is sound](https://github.com/sassafrastech/madeline_system/pull/20#discussion_r49797743)